vscripts/client/cl_fullscreen_map.nut

```diff
@@ -54,8 +56,22 @@ function FullscreenMinimapButtonReleased( player )
 	ResetAndShowMinimap( player )
 }
 
+function ToggleFullscreenMinimap( player )
+{
+	if ( !IsAlive( player ) || IsWatchingKillReplay() || !level.fullscreenMinimapEnabled || Riff_MinimapState() == eMinimapState.Hidden || 	PlayerHasPassive( player, PAS_MINIMAP_ALL ) )
+		return
+
+	if ( level.showingFullscreenMap )
+		ResetAndShowMinimap( player )
+	else
+		ShowFullMap( player )
+}
```

Comment:
```suggestion
function ToggleFullscreenMinimap( player )
{
	if ( !IsAlive( player ) || IsWatchingKillReplay() || !level.fullscreenMinimapEnabled || Riff_MinimapState() == eMinimapState.Hidden )
		return

	if ( level.showingFullscreenMap )
		ResetAndShowMinimap( player )
	else if ( !PlayerHasPassive( player, PAS_MINIMAP_ALL ) )
		ShowFullMap( player )
}
```
We want the check `PlayerHasPassive( player, PAS_MINIMAP_ALL )` to only apply before showing the map as in the original. Otherwise if the player suddenly gets the passive, they wouldn't be able to close the map that they have opened before anymore and they'd be stuck with it open